service/src/test/kotlin/no/nav/su/se/bakover/service/søknad/NySøknadTest.kt

```diff
@@ -100,8 +102,9 @@ class NySøknadTest {
         val personOppslagMock: PersonOppslag = mock {
             on { person(any()) } doReturn person.right()
         }
+        val sak = sak.copy()
```

Comment:
Sak er nå immutable (kun vals), så en trenger ikke beskytte seg for mutering lenger i testene. Tror det bare er Behandling som er mutable :question:

Reply:
True, fixar det!